net-p2p/bitcoind/bitcoind-0.16.0.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 2010-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DB_VER="4.8"
+inherit autotools bash-completion-r1 db-use systemd user
+
+MyPV="${PV/_/}"
```

Comment:
Please don't use camelcase in ebuilds, it's inconsistent with other variables you have there.